server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -999,6 +1002,12 @@ private TemplateVariables buildTemplateVariables(CreateFlowInstanceReq flowInsta
         return variables;
     }
 
+    private void setDefaultSchemaIfNeed(CreateFlowInstanceReq req, ConnectionConfig config) {
+        if (req.getTaskType() == TaskType.ASYNC && config.getDialectType() == DialectType.ORACLE) {
+            config.setDefaultSchema(databaseService.detail(req.getDatabaseId()).getName());
```

Comment:
Here only database name is used, so better to use `nullSafeGet()` to replace `detail()` to avoid unnecessary DB query.

Reply:
Get～

Reply:
This method has been deleted.